app/forms/provider_interface/offer_wizard.rb

```diff
@@ -54,5 +102,15 @@ def last_saved_state
     def state
       as_json(except: %w[state_store errors validation_context course_option]).to_json
     end
+
+    def update_path_history(attrs)
```

Comment:
maybe this can be extracted outside this wizard and into its own class to deal with the back link functionality that can be reused/tested in isolation?